server/src/main/java/org/apache/iotdb/db/qp/physical/sys/DataAuthPlan.java

```diff
@@ -49,6 +49,11 @@ public DataAuthPlan(OperatorType operatorType, List<String> users) {
     return null;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    return null;
```

Comment:
`Collections.emptyList()`

Reply:
I'll change it. Thanks for your suggestion.